docs/framework/wcf/feature-details/stand-alone-json-serialization.md

```diff
@@ -116,7 +116,7 @@ The data contract versioning features, including the <xref:System.Runtime.Serial
 
 When using ASP.NET AJAX endpoints with the HTTP GET verb (using the <xref:System.ServiceModel.Web.WebGetAttribute> attribute), incoming parameters appear in the request URL instead of the message body. JSON is supported even in the request URL, so if you have an operation that takes an `Int` called "number" and a `Person` complex type called "p", the URL may resemble the following URL.
 
-```
+```javascript
```

Comment:
Don't we have some identifiers for URLs that we can use? I can't recall right now.

Reply:
Here are a few that could apply:
The example is about including JSON in a Request URL so we could use `json` since that is what is being called out.
The Request URL is being fed to a ASP.NET endpoint, so we could use `aspx-csharp` to emphasize that.
For URLs we have `html` and `cshtml`.

`javascript` is definitely wrong (sorry about that :)) - my vote is for `json`.

Reply:
Let's do html then.

Reply:
@mairaw Thanks! I made the change.